app/Core/Mail/Client.php

```diff
@@ -44,16 +44,16 @@ public function __construct(Container $container)
      * @param  string  $html
      * @return Client
      */
-    public function send($recipientEmail, $recipientName, $subject, $html)
+    public function send($recipientEmail, $recipientName, $subject, $html, $authorName=null, $authorEmail=null)
     {
         if (! empty($recipientEmail)) {
             $this->queueManager->push(EmailJob::getInstance($this->container)->withParams(
                 $recipientEmail,
                 $recipientName,
                 $subject,
                 $html,
-                $this->getAuthorName(),
-                $this->getAuthorEmail()
+                (is_null ($authorName) ? $this->getAuthorName() : $authorName),
```

Comment:
The first and last parentheses are not necessary. Can you change `is_null (` to `is_null(`?